src/index.js

```diff
@@ -1,41 +1,104 @@
 import './style.css';
+import tasksRefactor from '../modules/TaskHandle.js';
 
-const todos = [
-  {
-    title: 'Watch the Tvs',
-    completed: false,
-    index: 1,
-  },
-  {
-    title: 'Read Js books',
-    completed: false,
-    index: 2,
-  },
-  {
-    title: 'Play the Sudoku',
-    completed: false,
-    index: 3,
-  },
-  {
-    title: 'Complete Weekly Project for the week',
-    completed: false,
-    index: 4,
-  },
-  {
-    title: 'work on Personal soft skilss',
-    completed: false,
-    index: 5,
-  },
-];
+const newTodoForm = document.querySelector('#form-sec');
+const todoList = document.querySelector('.task-section');
+const removeMarked = document.querySelector('.clearComplete');
+
+const updateInputText = (id, newText) => {
+  const todoListArray = JSON.parse(localStorage.getItem('todos') || '[]');
+  const targetIndex = todoListArray.findIndex((todo) => todo.id === parseInt(id, 10));
+
+  if (targetIndex >= 0) {
+    todoListArray[targetIndex].description = newText;
+    localStorage.setItem('todos', JSON.stringify(todoListArray));
+    tasksRefactor(todoListArray);
+  }
+};
+
+const toggleComplete = (id) => {
+  const todoListArray = JSON.parse(localStorage.getItem('todos') || '[]');
+  const updateTodoList = todoListArray.map((todo) => {
+    if (todo.id === parseInt(id, 10)) {
+      return { ...todo, completed: !todo.completed };
+    }
+    return todo;
+  });
+
+  tasksRefactor(updateTodoList);
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+};
+
+const removeTodo = (targetIndex) => {
+  const todoListArr = JSON.parse(localStorage.getItem('todos') || '[]');
+  const updateList = todoListArr.filter((todo) => todo.id !== parseInt(targetIndex, 10));
+  tasksRefactor(updateList);
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+};
+
+const removeCompleted = () => {
+  const updtArr = [];
+  const todoListArray = JSON.parse(localStorage.getItem('todos') || '[]');
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const todo of todoListArray) {
+    if (!todo.completed) {
+      updtArr.push(todo);
+    }
+  }
```

Comment:
- [ ] Please I highly suggest implementing the "Clear all completed" with .filter method in your functionality per the project requirement. 😄

This is a screenshot of the project requirements.

![Screenshot (286)](https://user-images.githubusercontent.com/78104544/236590635-e287f90e-5806-4b8a-8b13-f7e4f8840b9d.png)